recipes/pytest-pycodestyle/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pytest-pycodestyle" %}
+{% set version = "2.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e755e4235ed399760257026b8c3c696520848cb0261277c2df1c8e64af979eb2
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >3
```

Comment:
The python version looks like it should be `python >=3.6,<4` under `host` and `run`. Also `pytest >=5.4`

Reply:
Thank you for reviewing it. Done.